Generators/src/GeneratorFromFile.cxx

```diff
@@ -29,6 +29,13 @@ namespace eventgen
 {
 GeneratorFromFile::GeneratorFromFile(const char* name)
 {
+  if (strncmp(name, "alien:/", 7) == 0) {
```

Comment:
I think GeneratorFromFile is meant to read from Run2 AliRoot kinematics files. Probably the changes here do not make sense for these files (we can concentrate on the GeneratorFromO2Kine case).

Reply:
Done